packages/router/package.json

```diff
@@ -15,13 +15,20 @@
   "scripts": {
     "build": "ember build",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
     "start": "ember serve",
     "test": "ember test --test-port=0",
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.1.2"
+    "@ember/test-helpers": "^1.5.0",
+    "@embroider/compat": "^0.3.3",
+    "@embroider/macros": "^0.3.3",
+    "@embroider/webpack": "^0.3.3",
+    "ember-cli": "^3.9.0",
+    "ember-cli-babel": "^7.1.2",
+    "ember-qunit": "^4.4.1",
+    "ember-source-channel-url": "^1.1.0",
+    "qunit": "^2.9.2"
```

Comment:
This is not correct. We cannot have `dependencies` on these things, they're really supposed to be devDependencies.

Presumably the linter told you to do this, but that's because some override got dropped that tells eslint that certain folders contain tests and are allowed to use devDependencies.

I think eslint also respects .npmignore, so if you make sure the tests are in .npmignore that would also address the spurious lint errors.

Reply:
Yeah, doesn't make sense to have `qunit` there, should have noticed it. I'll look into this.